src/scripts/records/VideoRecords.js

```diff
@@ -19,8 +19,11 @@ class Video extends ImmutableRecord({
   title: '',
   author: '',
   free: '',
-  // published: '',
   staked: new StakingRecord(),
+  whiteListed: null,
+  challengeExists: null,
+  vote: null,
```

Comment:
Should these defaults be `null` or `false`?

Also, just checking are these new fields going to be present in `paratii-db` as well?